packages/create-bison-app/template/components/auth/LoginForm.tsx

```diff
@@ -0,0 +1,133 @@
+import {
```

Comment:
<img width="427" alt="Screenshot 2022-11-02 at 3 50 30 PM" src="https://user-images.githubusercontent.com/6558157/199589014-961e8578-459d-4ee8-a685-f003b9d00bdb.png">

Eventually we'll want to make this UI look better in dark mode, though I'm not considering that necessary for this PR.

Reply:
Yea, I need to get a PR rolling with Chakra Theme files -- but I'm waiting on some other conversations to land first. 

@alexanderson1993 😉

Reply:
I'll be in your DMs

Reply:
Yes, I noted this and felt the same about it - I did not want to include it in this PR. I will create PR when this PR gets merged in.